cluster-controller/src/main/java/io/strimzi/controller/cluster/Main.java

```diff
@@ -52,7 +52,7 @@ static CompositeFuture run(Vertx vertx, KubernetesClient client, Map<String, Str
         EndpointOperations endpointOperations = new EndpointOperations(vertx, client);
 
         boolean isOpenShift = Boolean.TRUE.equals(client.isAdaptable(OpenShiftClient.class));
-        KafkaClusterOperations kafkaClusterOperations = new KafkaClusterOperations(vertx, isOpenShift, configMapOperations, serviceOperations, statefulSetOperations, pvcOperations, podOperations, endpointOperations);
+        KafkaClusterOperations kafkaClusterOperations = new KafkaClusterOperations(vertx, isOpenShift, configMapOperations, serviceOperations, statefulSetOperations, pvcOperations, podOperations, endpointOperations, deploymentOperations);
```

Comment:
It was a crazy long line before, but could we fix that?

Reply:
It is not only quite long, but adding a new operation requires quite a lot of changes through all tests etc.